pkgs/applications/misc/xkbd/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, freetype, libXrender, libXft, xextproto
+, xinput, libXi, libXext, libXtst, libXpm, libX11, xproto, autoconf
+, automake, libtool, pkgconfig
+}:
+
+stdenv.mkDerivation rec {
+  name = "xkbd-${version}";
+  version = "0.8.18";
+
+  src = fetchFromGitHub {
+    owner = "mahatma-kaganovich";
+    repo = "xkbd";
+    rev = name;
+    sha256 = "05ry6q75jq545kf6p20nhfywaqf2wdkfiyp6iwdpv9jh238hf7m9";
+  };
+
+  buildInputs = [
+    freetype libXrender libXft libXext libXtst libXpm libX11
+    libXi xextproto xinput xproto
+  ];
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    libtool
+    pkgconfig
+  ];
+
+  preConfigure = ''
+    aclocal
```

Comment:
Would not autoreconf work here instead of all the commands?

Reply:
perharps but i followed the manual... let me keep that for next release.

Reply:
i'm in a "Release early, release often" , i improve navit derivation and need that. I always keep a lot of improvement in a zim wiki, and i improve step by step.

Reply:
Ah right, you can just set `nativeBuildInputs = [ autoreconfHook ];` and remove `preConfigure` and it will work (tested this myself).